src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx';
+import React, { FC, useState } from 'react';
+import { HexColorPicker } from 'react-colorful';
+
+import './ColorPicker.scss';
+import { ColorPickerProps } from './ColorPicker.types';
+
+const ColorPicker: FC<ColorPickerProps> = ({
```

Comment:
We werken in alle componenten met de `variants` prop voor styling, dat geeft de afnemer een makkelijke manier om extra classNames toe te voegen.